src/main/java/md2k/mCerebrum/cStress/features/ECGFeatures.java

```diff
@@ -385,11 +383,12 @@ private void filterPeaksTemp2(DataPointStream Rpeak_temp2, DataPointStream Rpeak
             }
 
             for (int j = 0; j < diffRpeak.size(); j++) {
-                if (diffRpeak.get(j).timestamp < (AUTOSENSE.RPEAK_INTERPEAK_MULTIPLIER * frequency)) {
+                if (diffRpeak.get(j) < (AUTOSENSE.RPEAK_INTERPEAK_MULTIPLIER*1000)) {
```

Comment:
Try to avoid constants.  Was this supposed to be frequency?  or 1000?  If 1000, please add it to the AUTOSENSE.java constants list.